config/locales/en/sdg.yml

```diff
@@ -439,3 +439,12 @@ en:
       hint: "You can introduce the code of a specific goal/target or a text to find one"
       placeholder: "Write a goal or target code or description"
       remove_tag: "Remove"
+    stats:
```

Comment:
I wonder whether we should move these lines so they're next to the rest of the `stats` translations :thinking:. Right now it makes sense to me, since translations for budgets stats are there and not in the `budgets.yml` file. What do you think?

Reply:
Ok, as I am going to change the component name as you suggested, `admin.yml` would be a better location for the component translations and is consistent with other stats translations.